app/boards/shields/zodiark/zodiark.conf

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Uncomment the following line to enable the Zodiark OLED Display
+# CONFIG_ZMK_DISPLAY=y
+
+# Uncomment these two lines to add support for encoders
+# CONFIG_EC11=y
+# CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+# CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
```

Comment:
```suggestion
```
I don't think this needs to be in here, as it's a general debug config option non-specific to this shield 👍